README.md

```diff
@@ -18,40 +18,59 @@ Tested with a standard 3rd party `oauth2_provider` Django application from
 
 Supported Versions
 ------------------
-Python 2.7 and Django 1.11,
-Python 3.7 and Django 2.2
+- Python 2.7 and Django 1.11
+- Python 3.7 and Django 2.2
+- PostgreSQL >= 9.6.9
+
+Other versions may likely work too, but are not supported.
 
 Python and Django compatibility is maintained in `compat.py` and `test_compat.py`
 
-Quick Start
------------
+### Installation
```

Comment:
could you also clarify which grant types are available out of the box w/o need in extras?

Reply:
it is clarified in lines 38 and 39